src/AasxServerDB/EditDB.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AasCore.Aas3_0;
+using AdminShellNS;
+using Microsoft.EntityFrameworkCore;
+
+namespace AasxServerDB
+{
+    public class EditDB
+    {
+        static public void EditAAS(AdminShellPackageEnv env)
+        {
+            Dictionary<string, int> aasToDeleteAASXIdsDic; 
+
+            var aasList = env.AasEnv.AssetAdministrationShells;
+            var aasIds = aasList.Select(x => x.Id).ToList();
+
+            using (AasContext db = new AasContext())
+            {
+                // Delets automtically from DB
+                aasToDeleteAASXIdsDic = db.AASSets.Where(x => aasIds.Contains(x.Identifier)).ToDictionary(x => x.Identifier, x => x.AASXId);
+                db.AASSets.Where(x => aasIds.Contains(x.Identifier)).ExecuteDelete();
+
+                // Load Everything back in

```

Comment:
Why do you need to delete every entry and then save them again? I would expect that you could search for each entry if it already exists and then updates or creates it. Deleting all of them first seems like a mistake